services/geocoding.py

```diff
@@ -27,15 +32,22 @@ def enhance(self, entries: List[PointOfInterest]) -> List[PointOfInterest]:
         For entries with missing lat/lng but present address
```

Comment:
The problem is that you didn't even try to make the injection, you need to have:

```
    # Injected dependencies
    settings: Settings
    make_geocode_request: Callable = make_geocode_request
```

Then you'll be able to actually swap it

And then you need to use:

```
geoData = self.make_geocode_request(address, gmaps)[0]["geometry"]["location"]
```